exonum/src/node/consensus.rs

```diff
@@ -250,7 +251,7 @@ where
     /// Executes and commits block. This function is called when node has full propose information.
     pub fn has_full_propose(&mut self, hash: Hash, propose_round: Round) {
         // Send prevote
-        if self.state.locked_round() == 0 {
+        if self.state.locked_round().0 == 0 {
```

Comment:
maybe as method `next` in `Height`, you could implement static method `Round::zero()`, and keep internal value private?